source/PreferencesPanel.cpp

```diff
@@ -506,7 +509,7 @@ void PreferencesPanel::DrawSettings()
 		"Show escort systems on map",
 		"Show stored outfits on map",
 		"System map sends move orders",
-		"Warning siren"
+		ALERT_INDICATOR
```

Comment:
visual stuff is further up

Reply:
This setting is not exclusively display related.